Cutelyst/Plugins/Session/sessionstorefile.cpp

```diff
@@ -74,10 +74,22 @@ bool SessionStoreFile::deleteSessionData(Context *c, const QString &sid, const Q
     return true;
 }
 
+QString rootPath()
+{
+    return QDir::tempPath()
+            + QLatin1Char('/')
+            + QCoreApplication::applicationName()
+            + QLatin1String("/session/data");
+}
+
 bool SessionStoreFile::deleteExpiredSessions(Context *c, quint64 expires)
 {
     Q_UNUSED(c)
-    Q_UNUSED(expires)
+    if(!expires)
```

Comment:
minor style nip tick the braces are kept on the if line, and the if<space>()<space>{